frontend-react/src/components/Table/Pagination.test.tsx

```diff
@@ -9,55 +9,38 @@ import Pagination, {
 } from "./Pagination";
 
 describe("Pagination", () => {
-    // Text match for getting pagination elements by role to capture previous
-    // and next links, page numbers, and overflow indicators (which are for
-    // presentation).
-    const ITEM_ROLE = /listitem|presentation/i;
-
     test.each([
         {
             description: "on the first page of an unbounded set",
             slots: [1, 2, 3, 4, 5, 6, OVERFLOW_INDICATOR],
             currentPageNum: 1,
-            expectedItems: ["1", "2", "3", "4", "5", "6", "…", "Next"],
         },
         {
             description: "on the second page of an unbounded set",
             slots: [1, 2, 3, 4, 5, 6, OVERFLOW_INDICATOR],
             currentPageNum: 2,
-            expectedItems: [
-                "Previous",
-                "1",
-                "2",
-                "3",
-                "4",
-                "5",
-                "6",
-                "…",
-                "Next",
-            ],
         },
         {
             description: "on the first page of a bounded set",
             slots: [1, 2],
             currentPageNum: 1,
-            expectedItems: ["1", "2", "Next"],
         },
         {
             description: "on the last page of a bounded set",
             slots: [1, 2],
             currentPageNum: 2,
-            expectedItems: ["Previous", "1", "2"],
         },
         {
             description: "when there is only one page",
             slots: [1],
             currentPageNum: 1,
-            expectedItems: ["1"],
         },
     ])(
         "Handles Previous and Next links $description",
-        ({ slots, currentPageNum, expectedItems }) => {
+        ({ slots, currentPageNum }) => {
```

Comment:
I feel like the changes here are making it harder to understand exactly what these tests are validating at first glance.  I realize that the custom logic here is to account for the different descendants (and that we also chatted about the direct descendants in the previous pull request), so this might be a good use case for using a common `data-testid` between all of them just so we can grab all buttons and overflow indicators and run expectations against their text content.  What do you think?